source/develop/release-management/features/XenToOvirt.html.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Importing Xen on REHL 5.x to oVirt
+category: feature
+authors: Shahar Havivi
+wiki_category: Feature|v2v
+wiki_title: Features/XenToOvirt
+wiki_revision_count: 1
+wiki_last_updated: 2015-06-02
+feature_name: 'v2v: Importing Xen on RHEL to oVirt'
+feature_modules: all
+---
+
+# Importing Xen on RHEL 5.x VMs to oVirt
+-----------------------------------------------------------
+### Summary
+oVirt had the ability to import VMs from other hypervisor including **Xen** on RHEL 5.x (not yet for Citrix Xen)
```

Comment:
had -> has ? or perhaps 'will have', as this is a the feature description?

RHEL 5 -> EL 5? (as this is upsteream)